assets/erc-7291/contracts/ERC1155Metadata_URI.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Code included in this repository should be released CC0-1.0, and must be released under a permissive license that doesn't impose too much on downstream implementations. In short, CC0-1.0 is preferred, but MIT/Apache-2.0 is allowed.

Reply:
Licenses of sample code have been changed to CC0-1.0